.github/workflows/ci.yml

```
@@ -17,13 +17,24 @@ jobs:
             os: macos-12
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+
+      - name: Install dependencies (Linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y pkg-config libfreetype6-dev libcairo2-dev
+
+      - name: Install dependencies (macOS)
+        if: matrix.os == 'macos-12'
```

Update to fix getting an `available runner`

```suggestion
        if: matrix.os == 'macos-13'
```

See https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#viewing-available-runners-for-a-repository